core/src/test/java/com/linecorp/armeria/server/ServerTlsTest.java

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright 2019 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.armeria.server;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import java.security.KeyStore;
+
+import javax.net.ssl.KeyManagerFactory;
+import javax.net.ssl.TrustManagerFactory;
+
+import org.junit.jupiter.api.Test;
+
+import com.linecorp.armeria.common.HttpResponse;
+import com.linecorp.armeria.common.HttpStatus;
+
+class ServerTlsTest {
+
+    @Test
+    void testJksKeyStoreWithNullPassword() throws Exception {
```

Comment:
How about adding `@Ignore` (with comments on why) and making this test case verify if an exception is raised just like `testPkcs12KeyStoreWithNullPassword` does?

Reply:
Updated with comments. PTAL :bowing_man: